Dockerfile-linux.template

```
@@ -332,6 +332,7 @@ RUN set -eux; \
 		--with-openssl \
 		--with-readline \
 		--with-zlib \
+		--with-iconv=shared \
```

These should be alphabetical: :sweat_smile: :eyes:

```suggestion
		--with-iconv=shared \
		--with-openssl \
		--with-readline \
		--with-zlib \
```

---

Yeah, they probably should be. I mean some of the surrounding options are a little out of order, but I'll make the change.